h2oai/h2oai.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "h2oai",
+    "key": "h2oai",
+    "label": "H2O AI",
+    "description": "AI for the Digital Brain",
+    "info": "https://www.h2o.ai/",
+    "logo": "https://h2o2016.wpengine.com/wp-content/themes/h2o2016/images/H2O_logo_yellow.svg",
+    "image": {
+        "registry" : "",
+        "name" : "h2oai",
+        "tags" : [ "latest" ]
+    },
+    "display": "stack",
+    "access": "external",
+    "description": "Description of my service",
+    "depends": [],
+    "config":  [],
+    "ports": [
+        {
+            "port": 54321,
+            "protocol": "http"
+        }
+    ],
+    "repositories":  [
+        {
+            "url": "https://raw.githubusercontent.com/h2oai/h2o-3/master/Dockerfile",
+            "type": "git"
+        }
+    ],
+    "developerEnvironment" : "",
+    "volumeMounts":[],
```

Comment:
I'm surprised that there are no volume mounts (i.e., no data stored by the tool in the container).  As far as I can tell, data/notebooks must be uploaded/downloaded to local computer.  Might be worth confirming.